.github/workflows/rebuild-schema.yml

```diff
@@ -28,7 +28,6 @@ jobs:
           title: 'Client auto update'
           body: |
             Underlying schema has changed. Check changes and approve accordingly. This will not create any release, simply assimilate changes on the schema.
-            version change this is.
```

Comment:
I don't think this belongs in the same PR. It doesn't fix anything, just removes text